### PR TITLE
running docker inside WSL page created

### PR DESCRIPTION
**Pull request by Henokaa**
I created a more explanatory page on the running docker page that describes how to install WSL2, set up ubuntu for windows users, and installation of docker in windows operating system.
So that window user won't have problems setting up ubuntu and all the additional steps needed for windows to install BookBrainz locally.

https://user-images.githubusercontent.com/69524541/161388796-85215dc5-ee30-4da3-ae95-f0e1773b2379.mp4

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -67,7 +67,8 @@ You'll need to install Docker and Docker-compose on your development machine:
 *  `Docker-compose`_
 
 .. important:: 
-  We recommended Windows users to run `docker inside WSL <https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers>`_ (preferbly WSL2) to avoid any compatibility issues.
+  
+  We recommended Windows users to :doc:`docker-setup` (preferbly WSL2) to avoid any compatibility issues.
```
Comment (MonkeyDo) [inline docs/installation.rst:71]: "We recommended Windows users to running docker inside WSL" sounds a bit weird.
I see you set up auto-labeling but "to run docker inside WSL" would make more sense.
We can maybe change the header of docker-setup.rst ?

**Review by MonkeyDo — COMMENTED** [review]
Thanks a lot, that's a great improvement !

**Comment by Henokaa** [comment]
@MonkeyDo Thanks a lot for the feedback, I made the adjustments. Let me know what you think.

**Review by MonkeyDo — APPROVED** [review]
Thank you !

**Pull request merged**